src/sorcha_community_utils/activity/lsst_comet/model.py

```diff
@@ -0,0 +1,129 @@
+__all__ = ["Comet"]
+
+import numpy as np
+
+try:
+    import astropy.units as u
+except ImportError:
+    u = None
+
+from .phase import phase_HalleyMarcus
+
+# TODO: Change this to use the Astropy definition
+AU_IN_CM = 14959787070000
+
+
+class Comet:
+    """LSST comet model.
+
+    The goal of this model is to make a template comet based on ``H``
+    magnitude, which is the main brightness parameter in the LSST
+    Metrics Analysis Framework.  Using this value enables easy object
+    cloning.
+
+
+    Parameters
+    ----------
+    R : float
+        Absolute magnitude or radius (km) of the nucleus.
+
+    afrho1 or afrho_q: float
+        Comet coma quantity Afρ at 1 au (``afrho1``) or at perihelion
+        (``afrho_q``), in units of cm.  The latter requires ``q``.
+
+    k : float
+        Activity power-law slope with heliocentric distance: ``rh^k``.
+
+    q : float, required if ``afrho_q`` is provided
+        Perihelion distance for Afρ normalization.
+
+    Phi_c : function, optional
+        One-parameter function that takes phase angle in degrees and
+        returns the phase function for comae.  It is assumed that
+        Phi(0) = 1.0.  Default is to use the Halley-Marcus phase
+        function of Schleicher & Bair 2011.
+
+    Phi_n : function, optional
```

Comment:
We've gotten rid of phi_n since. think comment goes

Reply:
Done